plugins/amdgpu/amdgpu_plugin_topology.c

```diff
@@ -452,7 +452,7 @@ static int parse_topo_node_mem_banks(struct tp_node *node, const char *dir_path)
 
 	while ((dirent_node = readdir(d_node)) != NULL) {
 		char line[300];
-		char bank_path[300];
+		char bank_path[1024];
```

Comment:
Could you please apply these changes in the "criu/plugin: Support AMD ROCm Checkpoint Restore with KFD" commit, which introduced `amdgpu_plugin_topology.c`?
This could be done, for example, with `git rebase -i 311aee4ff^`, then change `pick` to `edit` on the first line and save.
```
edit 311aee4ff criu/plugin: Support AMD ROCm Checkpoint Restore with KFD
```
After the changes have been applied you can use `git commit -a --ammend` and `git rebase --continue`.

Reply:
> > could you please advise if this ( [f81a453](https://github.com/checkpoint-restore/criu/commit/f81a453095118cb0a8c8bcb44d4d3193657a6cfb) ) approach to fix some container build dependencies is OK ?
> 
> The build dependencies for amdgpu_plugin should be optional and CRIU should work even if `libdrm-dev` is not installed. As far as I know, the current CI doesn't have AMD GPU available to run tests for the plugin, but it would be good to have tests for it.

I am not sure If I understood you here.  I made the change to the CI docker scripts but the tests were not triggered. I am not sure if maintainers need to manually trigger those.

Reply:
Currently CRIU would fail to compile if `libdrm` is not installed. Instead, when `libdrm` is not installed it should skip building `amdgpu_plugin`

> I made the change to the CI docker scripts but the tests were not triggered.

The CI tests are triggered on git push, for example, you can see in the [Alpine test logs](https://github.com/checkpoint-restore/criu/pull/1519/checks?check_run_id=2948771205) that `libdrm-dev` has been installed:
```
Step 3/11 : RUN apk update && apk add 	$CC 	bash 	build-base 	coreutils 	git 	gnutls-dev 	libaio-dev 	libcap-dev 	libnet-dev 	libnl3-dev 	nftables 	nftables-dev 	pkgconfig 	protobuf-c-dev 	protobuf-dev 	py3-pip 	py3-protobuf 	python3 	sudo 	libdrm-dev
...
(59/113) Installing libdrm (2.4.106-r0)
(60/113) Installing libpciaccess-dev (0.16-r0)
(61/113) Installing libdrm-dev (2.4.106-r0)
...
```